spanner_orm/field.py

```diff
@@ -194,19 +206,26 @@ def validate_type(value) -> None:
 
   @staticmethod
   def matches(type: str) -> bool:
-    return type.startswith('STRING(') and type.endswith(')')
+    val = re.findall('STRING\((.*)\)', type)
+    # We expect exact one matching result if type matches STRING.
+    if not val or len(val) != 1:
+      return False
+    return val[0] == 'MAX' or val[0].isnumeric()
```

Comment:
(nit) This could be simplified by using `STRING\((?:[0-9]+|MAX)\)` as the regex.